src/Grid.js

```diff
@@ -23,6 +23,7 @@ var Grid = React.createClass({
     headerRows: PropTypes.oneOfType([PropTypes.array, PropTypes.func]),
     rowHeight: PropTypes.number,
     rowRenderer: PropTypes.func,
+    debounceRowRequestWait: PropTypes.number,
```

Comment:
we should also pass this in as prop from ReactDataGrid component https://github.com/adazzle/react-data-grid/blob/master/src/addons/grids/ReactDataGrid.js which is the main wrapper grid. 
The future plan is to merge this `Grid` and `ReactDataGrid` components to remove confusion